File: c#/src/Docmost.Sdk/Models/SearchSuggestionDto.cs
namespace Docmost.Sdk.Models;

public class SearchSuggestionDto
{
    [System.Text.Json.Serialization.JsonPropertyName("query")]
    public string Query { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("includeUsers")]
    public bool? IncludeUsers { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("includeGroups")]
    public bool? IncludeGroups { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("includePages")]
    public bool? IncludePages { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("spaceId")]
    public string? SpaceId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("limit")]
    public int? Limit { get; set; }

}

File: go/docmost/models/search_suggestion_dto.go
package models

// SearchSuggestionDto from OpenAPI.
type SearchSuggestionDto struct {
	Query string `json:"query"`
	IncludeUsers *bool `json:"includeUsers,omitempty"`
	IncludeGroups *bool `json:"includeGroups,omitempty"`
	IncludePages *bool `json:"includePages,omitempty"`
	SpaceId *string `json:"spaceId,omitempty"`
	Limit *int `json:"limit,omitempty"`
}
